app/models/family.rb

```diff
@@ -109,7 +109,7 @@ def net_worth
   end
 
   def assets
-    Money.new(accounts.active.assets.map { |account| account.balance_money.exchange_to(currency, fallback_rate: 0) }.sum, currency)
```

Comment:
In these two places, we will need to call `account.value.exchange_to(currency, fallback_rate: 0)` so that users with accounts in multiple currencies can be "rolled up" properly.